sdk/eng/mgmt/mgmtmetadata/compute_resource-manager.txt

```diff
@@ -0,0 +1,15 @@
+Installing AutoRest version: v2
+AutoRest installed successfully.
```

Comment:
what is this file for?

Reply:
it gets generated with SDK generation. Not to worry about. I will take it out if it needs be